src/main/java/io/curity/identityserver/plugin/events/listeners/AccessTokenIssuedListener.java

```diff
@@ -59,6 +60,7 @@ public AccessTokenIssuedListener(AWSEventListenerConfiguration configuration)
         _tableName = configuration.getDynamodbTableName();
         _keyColumn = configuration.getTokenSignatureColumn();
         _roleARN = configuration.getAwsRoleARN();
+        _useInstanceProfile = configuration.isUseEC2InstanceProfile();
```

Comment:
Maybe add `orElse(false)` and the field can be just a `boolean`. This will make the if check below a bit more concise.

Reply:
The configuration is now handled differently so this is no longer an option.